starky/src/cross_table_lookup.rs

```diff
@@ -949,7 +945,9 @@ pub fn verify_cross_table_lookups<F: RichField + Extendable<D>, const D: usize,
                 .iter()
                 .map(|&table| *ctl_zs_openings[table].next().unwrap())
                 .sum::<F>()
-                + extra_sum_vec[c];
+                // Get elements looking into `looked_table` that are not associated to any STARK.
+                + ctl_extra_looking_sums
+                .map(|v| v[index][c]).unwrap_or_default();
```

Comment:
Should this be:
```suggestion
                .map(|v| v[looked_table.table][c]).unwrap_or_default();
```
as in the circuit version? Or am I misunderstanding something?

Reply:
Yes, I think you are right.